rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -41,7 +41,49 @@ paths:
       parameters:
         - $ref: '#/components/parameters/Accept-Version'
       requestBody:
-        $ref: '#/components/requestBodies/IngestRequest'
```

Comment:
@lessarderic and I had a discussion about this. We both agree that we should not sacrifice API quality because this bug exists in the code generation. However, we are actively using this API and could use this fix sooner than later. We agree that we can wait to address the issue until we are ready to release. I have created https://github.com/connexta/ion-ingest-api/issues/5 to track this effort.